db-scripts/createdb.sql

```diff
@@ -65,7 +65,7 @@ create table if not exists equipment_rentals(
    price money not null,
    condition equipment_condition_enum not null default 'ready',
    PRIMARY KEY(user_id, item_id),
-   FOREIGN KEY(item_id) REFERENCES equipment_item(id) on update cascade,
+   FOREIGN KEY(item_id) REFERENCES equipment_item(id) on update cascade on delete cascade,
```

Comment:
let's make the on delete action to be 'set null'. also please change user_id to be nullable. the reason for this is because this table will keep the entire history of the rented items. so we don't want to delete records just because users were deleted

Reply:
done